RELEASING.md

```diff
@@ -1,92 +1,68 @@
-
-1. Identify the packages that are affected by the bug. Make the changes to those packages, merging to `main`, as quickly as possible.
-2. On your local machine, remove the `dev0` tags from the version number and increment the patch version number.
-3. On your local machine, update `CHANGELOG.md` with the date of the hotfix change.
-4. With administrator priviledges for PyPi, manually publish the affected packages.
-    a. Install [twine](https://pypi.org/project/twine/)
-    b. Navigate to where the `setup.py` file exists for the package you want to publish.
-    c. Run `python setup.py sdist bdist_wheel`. You may have to install [wheel](https://pypi.org/project/wheel/) as well.
-    d. Validate your built distributions by running `twine check dist/*`.
-    e. Upload distibutions to PyPi by running `twine upload dist/*`.
-5. Note that since hotfixes are manually published, the build scripts for publish after creating a release are not run.
+# Release instructions
+
+## Preparing a new major or minor release
+
+* Run the [Prepare release branch workflow](https://github.com/trask/repository-template/actions/workflows/prepare-release-branch.yml).
+  * Press the "Run workflow" button, and leave the default branch `main` selected.
+  * Review and merge the two pull requests that it creates
+    (one is targeted to the release branch and one is targeted to `main`).
+
+## Preparing a new patch release
+
+* Backport pull request(s) to the release branch.
+  * Run the [Backport workflow](https://github.com/trask/repository-template/actions/workflows/backport.yml).
+  * Press the "Run workflow" button, then select the release branch from the dropdown list,
+    e.g. `release/v1.9.x`, then enter the pull request number that you want to backport,
+    then click the "Run workflow" button below that.
+  * Review and merge the backport pull request that it generates.
+* Merge a pull request to the release branch updating the `CHANGELOG.md`.
+  * The heading for the unreleased entries should be `## Unreleased`.
+* Run the [Prepare patch release workflow](https://github.com/trask/repository-template/actions/workflows/prepare-patch-release.yml).
+  * Press the "Run workflow" button, then select the release branch from the dropdown list,
+    e.g. `release/v1.9.x`, and click the "Run workflow" button below that.
+  * Review and merge the pull request that it creates for updating the version.
+
+## Making the release
+
+* Run the [Release workflow](https://github.com/open-telemetry/opentelemetry-python/actions/workflows/release.yml).
+  * Press the "Run workflow" button, then select the release branch from the dropdown list,
+    e.g. `release/v1.9.x`, and click the "Run workflow" button below that.
+  * This workflow will publish the artifacts and publish a GitHub release with release notes based on the change log.
```

Comment:
@trask 
For some reason, the `publish` workflow, which is responsible for publishing to PyPi, and which usually runs automatically when a release is published was not run on my fork when testing. Were you able to get this to run automatically on your fork?

Reply:
oh good catch, I think the workflow isn't running b/c the release is being published via workflow using the default GITHUB_TOKEN, and github has some protections against workflows triggering other workflows to prevent infinite loops (which is one of the reasons why many of the workflows in the PR use the BOT_TOKEN instead of the default GITHUB_TOKEN.

is it ok if I embed the `publish` workflow inside the release workflow (after the release is published)? that would be the simplest fix

Reply:
I embedded the publishing into the release workflow. There are ways to componentize large workflows, but that also brings some complexity.